JASP-Common/importers/jaspimporter.cpp

```diff
@@ -28,8 +28,14 @@ void JASPImporter::loadDataSet(DataSetPackage *packageData, const string &path,
 
 	readManifest(packageData, path);
 
-	if ( ! isCompatible(packageData))
+	Compatibility compatibility = isCompatible(packageData);
+	if (compatibility == JASPImporter::NotCompatible)
 		throw runtime_error("The file version is too new.\nPlease update to the latest version of JASP to view this file.");
+	else if (compatibility == JASPImporter::Limited)
+		packageData->warningMessage = "WARNING: This file was created by a newer version of JASP and may not have complete functionality.";
```

Comment:
you could probably tone this down a bit; i.e. WARNING in capitals is a bit full on.